velox/dwio/parquet/writer/Writer.h

```diff
@@ -142,6 +143,8 @@ class Writer : public dwio::common::Writer {
   std::shared_ptr<ArrowContext> arrowContext_;
 
   std::unique_ptr<DefaultFlushPolicy> flushPolicy_;
+
+  TypePtr schema_;
```

Comment:
This feels like a requires parameter, not an option. Can we add it as an constructor argument instead?